tests/integration/up_down/test_podman_compose_up_down.py

```diff
@@ -0,0 +1,289 @@
+# SPDX-License-Identifier: GPL-2.0
+
+"""
+test_podman_compose_up_down.py
+
+Tests the podman compose up and down commands used to create and remove services.
+"""
+
+# pylint: disable=redefined-outer-name
+import os
+import unittest
+
+from tests.integration.test_utils import RunSubprocessMixin
+from tests.integration.test_utils import podman_compose_path
+from tests.integration.test_utils import test_path
+
+
+class TestPodmanCompose(unittest.TestCase, RunSubprocessMixin):
+    up_cmd = [
+        "coverage",
+        "run",
+        podman_compose_path(),
+        "-f",
+        os.path.join(test_path(), "up_down", "docker-compose.yml"),
+        "up",
+        "-d",
+        "--force-recreate",
+    ]
+
+    container_exists_cmd = [
```

Comment:
These commands are simple enough that they can be inlined. Right now we make tests a bit harder to understand.